kernel/Kconfig

```diff
@@ -157,7 +157,7 @@ config MAIN_STACK_SIZE
 	int "Size of stack for initialization and main thread"
 	default 2048 if COVERAGE_GCOV
 	default 1024 if TEST_ARM_CORTEX_M
```

Comment:
should be able to get rid of this now that ARM is no longer an exception and the default is 1024.

Reply:
I think yes, will remove it